src/decisionengine/framework/engine/Workers.py

```diff
@@ -47,30 +47,42 @@ def get_consumes(self):
         return self.task_manager.get_consumes()
 
     def run(self):
-        logger = logging.getLogger()
-        logger.setLevel(logging.WARNING)
+
+        myname = self.task_manager.name
+        myfilename = os.path.join(os.path.dirname(self.logger_config["log_file"]), myname + ".log")
+
+        logger = structlog.getLogger(f"{myname}")
+        logger.setLevel(logging.DEBUG)
```

Comment:
Shouldn't the loglevel come out of the config rather than be hard coded?

Reply:
The level IS set by the config value in line 87. I was being overly cautious by setting an initial default value for the logger in line 55. The default value for the non-root logger is NOTSET when the logger is created, and everything gets processed. However, if this behavior changes in the future and someone puts log in-between lines 55 and 87, the log might not be processed. I am fine taking out line 55, if you think it is cleaner.

Reply:
Ahh, any chance to drop a comment in the code just so folks (like me) know that we are setting a default until the config comes in.

Reply:
Will do.